views/house_wikidata.erb

```diff
@@ -18,3 +19,26 @@
       </ol>
   </div>
 </div>
+
+<script type="text/javascript">
+  $(document).ready(function() {
+    var wikidata_id = '<%= @page.wikidata_id %>',
+        endpointUrl = 'https://query.wikidata.org/sparql',
+        sparqlQuery = "SELECT ?seat_count\n" +
+                      "WHERE { wd:" + wikidata_id + " wdt:P1342 ?seat_count }",
+        settings = {
+          headers: { Accept: 'application/sparql-results+json' },
+          data: { query: sparqlQuery }
+        };
+
+    $.ajax(endpointUrl, settings).done(function(data) {
+      if (data.results.bindings[0]) {
+        var seat_count = data.results.bindings[0][data.head.vars[0]].value;
```

Comment:
I'm a little worried about silently only taking the first value if there's more than one response. For current, national-level legislatures, the wdt: _should_ be enough to always get the up-to-date value, but I'm not sure that's always going to be true, especially as we include some historic legislatures where what the truthiest value is is less well-defined. 

I'm not sure what to suggest instead, but as this is for an internal-only page to start with, simply displaying _all_ the returned values might be enough to start with, and will let us get an idea of when (if at all) we _do_ tend to get multiple values.